### PR TITLE
Form and Incident Reports page stylings

### PR DESCRIPTION
**Pull request by nickcabuno**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] `npm test`

# Checklist

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My code has been reviewed by at least one peer
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes
- [x ] There are no merge conflicts

### CODE REVIEW

**Review by jossters — APPROVED** [review]
Great Work on keeping a consistent style that conforms with the rest of the project.

File: src/components/AdminDashboard/Incidents.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const DashboardIncidents = props => {
     setPageNumber,
     unapprovedIncidents,
     setSelected,
-    formResponses
+    formResponses,
   } = props;
```
Comment (JAaron93) [inline src/components/AdminDashboard/Incidents.js]: Nice catch with this comma

File: src/components/form/TwitterForm.js [hunk 2]
```diff
@@ -88,120 +89,151 @@ const TwitterForm = () => {
...
-          Date
-        </label>
-        {/* Potential project for upcoming cohort, we ran out of time to figure out how defaultValue works for DatePicker from antd
+        <br></br>
+        <label htmlFor="incident_date" className="form-labels">
+          <b>Date</b>
+          <br></br>
+          {/* Potential project for upcoming cohort, we ran out of time to figure out how defaultValue works for DatePicker from antd
         Somehow, we have to be able to convert the "zulu timestamp" to a regular date statement YYYY-MM-DD. Since we were unable
         to do this, we've decided to keep the date blank for our submission. */}
```
Comment (JAaron93) [inline src/components/form/TwitterForm.js:204]: Love the documentation you have here for future cohorts. Surprised you guys weren't able to convert the Zulu time delta though.

File: src/components/AdminDashboard/FormResponseIncidents.js [hunk 3]
```diff
@@ -1,16 +1,10 @@
 import React, { useState, useEffect } from 'react';
```
Comment (devin-mitchell) [inline src/components/AdminDashboard/FormResponseIncidents.js:1]: Not sure this components is being used.  Looks like a shell of a component that somehow got merged previously.

**Comment by n0tybynature** [comment]
Good job nick, love the styling and consistent design.

**Review by n0tybynature — APPROVED** [review]
looks good. God job nick

**Pull request merged**